GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/Interpolator.java

```diff
@@ -0,0 +1,23 @@
+package org.gearvrf.x3d;
+
+public class Interpolator {
```

Comment:
Critical I save the Interpolator data.  There are several interpolators in X3D (position, orientation, scale, color).  They are declared anywhere (above the ROUTE that references them) in the .x3d file.  Only till the ROUTE references them some point later do I know which object is using the Interpolator. 
Is the issue that it's public instead of private?  I will research that.